kali/Vagrantfile

```diff
@@ -0,0 +1,173 @@
+#                     Copyright H Dub (hdub-tech) 2024
+#        Distributed under the Boost Software License, Version 1.0.
+# See accompanying file LICENSE or copy at https://www.boost.org/LICENSE_1_0.txt
+#                     Kali template - Version 1.0
+#
+# ####################################################################### #
+#                         UPDATE THIS BLOCK BELOW                         #
+# ####################################################################### #
+$vm_name = "vagrant-kalilinux-rolling"
+$hostname = "kali"
+
+# Only tested with bash/zsh
+$preferred_shell = "bash"
+
+# The folder on your system to share at /vagrant in the VM
+$local_shared_dir = "."
+
+# If false, the shell provisioners will run as the vagrant user
+$run_as_root = false
+
+# The version of Python to install with pyenv and set as the pyenv global default
+$python_version = "3.12"
+
+# These will be installed as root regardless of $run_as_root
+$apt_packages = [
+  "git",
```

Comment:
```suggestion
```

`git` was a requirement for using this Vagrantfile and moved to the provisioner - forgot to clean-up up top